src/services/json-api-datastore.service.ts

```diff
@@ -24,17 +21,16 @@ export class JsonApiDatastore {
   // tslint:disable-next-line:variable-name
   private _store: {[type: string]: {[id: string]: JsonApiModel}} = {};
   private toQueryString: Function = this.datastoreConfig.overrides
-    && this.datastoreConfig.overrides.toQueryString ?
-      this.datastoreConfig.overrides.toQueryString : this._toQueryString;
+  && this.datastoreConfig.overrides.toQueryString ?
```

Comment:
We should use a consistent style, so maybe you could undo the removing of the indention here?